src/main/java/application/controller/BookController.java

```diff
@@ -53,4 +54,12 @@ public BookDto updateBook(@PathVariable Long id, @RequestBody CreateBookRequestD
     public void deleteBookById(@PathVariable Long id) {
         bookService.deleteBookById(id);
     }
+
+    @GetMapping("/search")
+    public List<BookDto> getAllByInputs(BookSearchParametersDto bookSearchParametersDto) {

```

Comment:
You can use ResponseEntity class wrapper for controller return type. We would be able to control returning status code, for example